errors/errors.go

```diff
@@ -109,4 +109,5 @@ var (
 	ErrInvalidCertificateContent      = errors.New("signatures: invalid certificate content")
 	ErrInvalidStateCookie             = errors.New("auth: state cookie not present or differs from original state")
 	ErrSyncNoURLsLeft                 = errors.New("sync: no valid registry urls left after filtering local ones")
+	ErrNoGoModFileFound               = errors.New("test: no go.mod file found in parent directories")
```

Comment:
What is this error? inside zot?

Reply:
We decided to move this error under the pkg/test package.